test/oembed/test.ts

```diff
@@ -32,11 +32,15 @@ test("width/height should be numbers", async () => {
 
   const result = await unfurl("http://localhost/html/oembed");
 
-  expect(result.oEmbed.width).toEqual(640);
-  expect(result.oEmbed.height).toEqual(640);
+  expect(result.oEmbed?.type).toEqual("video");
+  const oEmbed =
+    result.oEmbed?.type === "video" ? result.oEmbed : (result.oEmbed as never);
```

Comment:
If the oEmbed type is "link", width and height do not exist, so it must first be checked that the oEmbed type is "video".

Also, with only `expect(result.oEmbed?.type).toEqual("video")`, TypeScript cannot infer that the type of `result.oEmbed` is `OEmbedVideo`. Therefore, the following type guard is additionally written.

```ts
const oEmbed = result.oEmbed?.type === "video" ? result.oEmbed : (result.oEmbed as never)
```

At this line, Since we have already checked that `result.oEmbed?.type === "video"` in the above line, the "else" part is regarded as `never`.